test/integration/operator_run_bundle_test.go

```diff
@@ -0,0 +1,235 @@
+// repository. The generated csv by default supports OwnNamespace, SingleNamespace and AllNamespaces.
+func createOperator(tc testutils.TestContext, projectName string) {
+	By("initializing a project")
+	err := tc.Init(
+		"--project-version", "3-alpha",
+		"--repo", path.Join("github.com", "example", projectName),
+		"--domain", tc.Domain,
+		"--fetch-deps=false")
+	Expect(err).Should(Succeed())
+
+	By("creating an API definition")
+	err = tc.CreateAPI(
+		"--group", tc.Group,
+		"--version", tc.Version,
+		"--kind", tc.Kind,
+		"--namespaced",
+		"--resource",
+		"--controller",
+		"--make=false")
+	Expect(err).Should(Succeed())
+
+	By("implementing the API")
+	Expect(kbtestutils.InsertCode(
+		filepath.Join(tc.Dir, "api", tc.Version, fmt.Sprintf("%s_types.go", strings.ToLower(tc.Kind))),
+		fmt.Sprintf(`type %sSpec struct {
+`, tc.Kind),
+		`	// +optional
+	Count int `+"`"+`json:"count,omitempty"`+"`"+`
+`)).Should(Succeed())
+
+	By("generating the operator bundle")
+	// Turn off interactive prompts for all generation tasks.
+	replace := "operator-sdk generate kustomize manifests"
+	testutils.ReplaceInFile(filepath.Join(tc.Dir, "Makefile"), replace, replace+" --interactive=false")
+	err = tc.Make("bundle", "IMG="+tc.ImageName)
+	Expect(err).NotTo(HaveOccurred())
+
+	By("building the operator bundle image")
+	// Use the existing image tag but with a "-bundle" suffix.
+	err = tc.Make("bundle-build", "BUNDLE_IMG="+tc.ImageName)
+	Expect(err).NotTo(HaveOccurred())
+
+	By("push the bundle to a remote repository")
+	err = tc.Make("docker-push", "IMG="+tc.ImageName)
+	Expect(err).NotTo(HaveOccurred())
+}
+
+func uninstall(kubeconfig string, tc testutils.TestContext) {
+	cfg := &operator.Configuration{KubeconfigPath: kubeconfig}
+	err := cfg.Load()
+	Expect(err).NotTo(HaveOccurred())
+
+	uninstallConfig := operator.NewUninstall(cfg)
+	uninstallConfig.DeleteAll = true
+	uninstallConfig.DeleteOperatorGroupNames = []string{tc.Group}
+	uninstallConfig.Package = defaultOperatorName
+	uninstallConfig.Logf = logrus.Infof
+
+	ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
+	defer cancel()
+
+	err = uninstallConfig.Run(ctx)
+	Expect(err).NotTo(HaveOccurred())
+
+	waitForConfigMapDeletion(ctx, cfg, defaultOperatorName)
+}
+
+func waitForConfigMapDeletion(ctx context.Context, cfg *operator.Configuration, packageName string) {
+	cfgmaps := corev1.ConfigMapList{}
+	opts := []client.ListOption{
+		client.InNamespace(cfg.Namespace),
+		client.MatchingLabels{"owner": "operator-sdk", "package-name": packageName},
+	}
+	err := wait.PollImmediateUntil(250*time.Millisecond, func() (bool, error) {
+		if err := cfg.Client.List(ctx, &cfgmaps, opts...); err != nil {
+			return false, err
+		}
+		return len(cfgmaps.Items) == 0, nil
+	}, ctx.Done())
+	Expect(err).NotTo(HaveOccurred())
+}
+
+func TestRunBundle(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Operator SDK run bundle test suite")
+}
```

Comment:
All integration tests in this package should be converted to ginkgo/gomega tests eventually like this integration test is, so the suite runner should be in a separate file and look like so:
```suggestion
func TestIntegration(t *testing.T) {
	RegisterFailHandler(Fail)
	RunSpecs(t, "Integration Suite")
}
```

Reply:
IMO we need to create the dir /test/e2e-olm/ in order to follow the project standard. The tests here are old and are not following the new definitions. Then, we should run `ginkgo bootstrap`. It will generated the files with the names following the lib convention. See: https://onsi.github.io/ginkgo/